src/fusor/harvester.py

```diff
@@ -44,11 +45,25 @@
 T = TypeVar("T", bound=Translator)
 
 
+@dataclass
```

Comment:
Consider using [Pydantic dataclass](https://docs.pydantic.dev/latest/concepts/dataclasses/) for validation